h2o-py/test-requirements.txt

```diff
@@ -35,7 +36,7 @@ lifelines>=0.19.5
 xlsxwriter==1.3.7
 xgboost==1.7.6
 causalml==0.12.3; python_version <= '3.7'
-causalml==0.14.1; python_version > '3.7'
+causalml==0.14.1; python_version > '3.7' and python_version < '3.11'
```

Comment:
should this be:

and python_version <= '3.11'

Reply:
Nope, causalml doesn't support python 3.11 yet.